.github/workflows/ci.yml

```diff
@@ -48,11 +48,13 @@ jobs:
           - 'ubuntu-1804'
           - 'ubuntu-2004'
         suite:
-          - 'package'
-          - 'binary'
-          - 'npm'
+          - 'default'
           - 'npm-embedded'
-          - 'npm-packages'
+          - 'package'
+          - 'source'
+        exclude:
+          - os: 'centos-7'
+            suite: 'source'
```

Comment:
It looks like this is due to the fact that the default compiler on EL7 isn't new enough to compile it correctly as noted here: https://github.com/nxhack/openwrt-node-packages/issues/470

A possible workaround would be to use the [devtoolkit package from SCL](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/), but that's probably better done in a separate PR using the `yum-centos` cookbook to enable the repository.

I recommend you put a comment here that explains why this is excluded and create an issue to get this properly fixed.

Reply:
Issue #218 already exists for tracking adding centos support.